### ISSUE TITLE
Option to sample new points

### ISSUE THREAD

**Issue by agosiewska** [OmaymaS/lime_explainer#1]
Great app! It would be nice to have an option to resample new data to see how the fitted line changes with new points.

**Comment by OmaymaS** [comment]
@agosiewska Yeah this is one option I had in mind as well. Thanks for highlighting it.
